docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -659,23 +576,23 @@ This code is using our spender client to do two things:
 1. calls `approveWithSignature` to approve the spend permission
 2. calls `spend` to make use of our allowance and spend our user's funds
 
-Since our spender is a smart contract wallet, notice that these calls are formulated as userOperations instead of 
-direct transactions. They'll be submitted to the blockchain by a bundler and the gas will be subsidized by our
-Coinbase Developer Platform paymaster.
 
 ### Try out your app
 
 Run your app locally with `bun run dev` and visit `localhost:3000`.
 
-You should see a "Connect wallet" button in the top right corner; click that.
+When you click the "Subscribe" button you should be prompted to create or connect your Smart Wallet.
 
 You can create a new Smart Wallet via the popup. Note that you'll need a little ETH in this wallet to fund the
 deployment of your account. If you don't have any testnet ETH, try this [Coinbase faucet](https://portal.cdp.coinbase.com/products/faucet).
 
-Once your wallet is created and funded, return to the app and click "Subscribe".
-Sign the prompts, which will deploy your new Smart Wallet (if undeployed), and then prompt you to sign over the spend permission.
+Before you continue, note that you'll also need a little testnet ETH in your spender wallet, which will need to fund the gas for the transactions.
```

Comment:
```suggestion
Note that we'll need a little bit of base sepolia ETH in both wallet addresses (the "user" wallet and the "app" wallet). In a more involved implementation you would use a paymaster to eliminate this requirement. For now, If you don't have any base sepolia ETH, try this [Coinbase faucet](https://portal.cdp.coinbase.com/products/faucet).
```

Do we actually require funds in the user wallet? Wouldn't the dapp's wallet be deploying the user's account during execution?

Reply:
iirc I've actually seen the popup prompt me to deploy the wallet.... though it's been a while since I remember that happening.  I just tried this with a fresh smart wallet and before I had any ETH i see this screen: 
![Uploading Screenshot 2024-11-07 at 6.57.36 PM.png…]()

Reply:
once I funded the wallet, I was not prompted to deploy the wallet by the popup, we moved right on to spendpermissions

Reply:
while I don't ever see the wallet prompt me to deploy it, I do see a warning screen if I have 0 ETH in the wallet with some buttons including the option to "Fund wallet".  So while the user may not require ETH in a functional sense, something in our system is enforcing that

Reply:
(ultimately they will require at least a little ETH if the dev wants to see the app work end-to-end because there needs to be a little for the spender to spend)

Reply:
Gotcha. If the dapp requests a base asset (i.e. ETH) then the user will be prompted to fund before granting (so the dapp has something to spend). If the dapp requests an erc20 in an ideal world we'd probably ask the user to fund there as well, but in the current world we just let the user grant the permission without funding.